pallets/funding/src/tests/2_evaluation.rs

```diff
@@ -0,0 +1,552 @@
+			let evaluations = vec![(EVALUATOR_1, target_evaluation_usd / 2).into()];
+			let evaluation_plmc = MockInstantiator::calculate_evaluation_plmc_spent(evaluations.clone());
+			let evaluation_existential = evaluation_plmc.accounts().existential_deposits();
+			inst.mint_plmc_to(evaluation_plmc);
+			inst.mint_plmc_to(evaluation_existential);
+
+			let project_id = inst.create_evaluating_project(project_metadata.clone(), ISSUER_2);
+			inst.evaluate_for_users(project_id, evaluations.clone()).unwrap();
+
+			let old_price = <TestRuntime as Config>::PriceProvider::get_price(PLMC_FOREIGN_ID).unwrap();
+			PRICE_MAP.with_borrow_mut(|map| map.insert(PLMC_FOREIGN_ID, old_price * 2.into()));
+
+			let update_block = inst.get_update_block(project_id, &UpdateType::EvaluationEnd).unwrap();
+			let now = inst.current_block();
+			inst.advance_time(update_block - now + 1).unwrap();
+			let project_status = inst.get_project_details(project_id).status;
+			assert_eq!(project_status, ProjectStatus::EvaluationFailed);
+		}
+	}
+
+	#[cfg(test)]
+	mod failure {
+		use super::*;
+
+		#[test]
+		fn round_fails_after_not_enough_bonds() {
+			let mut inst = MockInstantiator::new(Some(RefCell::new(new_test_ext())));
+			let now = inst.current_block();
+			let issuer = ISSUER_1;
+			let project_metadata = default_project_metadata(inst.get_new_nonce(), issuer);
+			let evaluations = default_failing_evaluations();
+			let plmc_eval_deposits: Vec<UserToPLMCBalance<_>> =
+				MockInstantiator::calculate_evaluation_plmc_spent(evaluations);
+			let plmc_existential_deposits = plmc_eval_deposits.accounts().existential_deposits();
+
+			let expected_evaluator_balances = MockInstantiator::generic_map_operation(
+				vec![plmc_eval_deposits.clone(), plmc_existential_deposits.clone()],
+				MergeOperation::Add,
+			);
+
+			inst.mint_plmc_to(plmc_eval_deposits.clone());
+			inst.mint_plmc_to(plmc_existential_deposits.clone());
+
+			let project_id = inst.create_evaluating_project(project_metadata, issuer);
+
+			let evaluation_end = inst
+				.get_project_details(project_id)
+				.phase_transition_points
+				.evaluation
+				.end
+				.expect("Evaluation round end block should be set");
+
+			inst.evaluate_for_users(project_id, default_failing_evaluations()).expect("Bonding should work");
+
+			inst.do_free_plmc_assertions(plmc_existential_deposits);
+			inst.do_reserved_plmc_assertions(plmc_eval_deposits, HoldReason::Evaluation(project_id).into());
+
+			inst.advance_time(evaluation_end - now + 1).unwrap();
+
+			assert_eq!(inst.get_project_details(project_id).status, ProjectStatus::EvaluationFailed);
+
+			// Check that on_idle has unlocked the failed bonds
+			inst.settle_project(project_id).unwrap();
+			inst.do_free_plmc_assertions(expected_evaluator_balances);
+		}
+	}
+}
+
+#[cfg(test)]
+mod start_evaluation_extrinsic {
+	use super::*;
+
+	#[cfg(test)]
+	mod success {
+		use super::*;
+
+		#[test]
+		fn evaluation_starts() {
```

Comment:
Think it might be good to check all the phase_transition_points that are set. See if they are "correct" as its the biggest change the do_start_evaluation should make.